src/test/libcephfs/snapdiff.cc

```diff
@@ -56,7 +59,58 @@ class TestMount {
     return ceph_conf_get(cmount, option, buf, len);
   }
 
-  string make_file_path(const char* relpath) {
+  int rank0_config_set(const char * var, const char * val) {
```

Comment:
minor nit: use `std::string` (or std::string_view`) for the parameter and use `.c_str()` to get the ptr ref - looks a bit more C++'y. Feel free to disagree too :)

Reply:
string view doesn't work because cmdmap_t doesn't like it. Otherwise, I've changed to string, though I'm not sure I could avoid all unnecessary copies.